# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: extensions/permissions/PermissionDelegateHandler.h
# Commit: a2bec965260d
# Full Hash: a2bec965260d58e8f5bd814c179444875d9962cf
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2020-06-13 03:21:52
# Regressor Bug: 1587743
# File Overlap Count: 1
# Description:
#   Bug 1587743 - Part 2: Removing mTopLevelPrincipal from the PermissionDelegateHandler. r=baku
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D79133
# ==============================================================================

diff -r 44e715808901 -r a2bec965260d extensions/permissions/PermissionDelegateHandler.h
--- a/extensions/permissions/PermissionDelegateHandler.h	Fri Jun 12 16:31:49 2020 +0000
+++ b/extensions/permissions/PermissionDelegateHandler.h	Fri Jun 12 16:31:56 2020 +0000
@@ -195,7 +195,6 @@
   mozilla::dom::Document* mDocument;
 
   nsCOMPtr<nsIPrincipal> mPrincipal;
-  nsCOMPtr<nsIPrincipal> mTopLevelPrincipal;
   RefPtr<nsIPermissionManager> mPermissionManager;
 };
 
